stream-api/src/main/kotlin/fund/cyber/markets/api/common/IncomingMessagesHandler.kt

```diff
@@ -17,8 +22,22 @@ class IncomingMessagesHandler(
         val command = commandsParser.parseMessage(bufferedMessage.data)
         when (command) {
             is UnknownCommand -> {}
+            is TradeChannelInfoCommand -> {
+                when (command.type) {
+                    IncomingMessageGetTopicType.PAIRS ->
+                        WebSockets.sendText(
+                                jsonSerializer.writeValueAsString(
```

Comment:
bad formating,
add extension method to jsonSerializer as ObjectMapper.serialize() : String  and use it to make code more readable